rpc/contract.go

```diff
@@ -39,6 +42,17 @@ func (h *Handler) StorageAt(address, key felt.Felt, id BlockID) (*felt.Felt, *js
 	}
 	defer h.callAndLogErr(stateCloser, "Error closing state reader in getStorageAt")
 
+	// This checks if the contract exists because if a key doesn't exist in contract storage,
+	// the returned value is always zero and error is nil.
+	_, err := stateReader.ContractNonce(&address)
```

Comment:
I think it would be better to use `ContractClassHash` to be more explicit as this is what we use in the [`deployed()`](https://github.com/NethermindEth/juno/blob/e4240acb490f6f213193459de66182ca24ff2810/core/contract.go#L82)

Reply:
changed